manifests/charts/base/values.yaml

```diff
@@ -13,4 +13,8 @@ global:
 
 base:
   # Used for helm2 to add the CRDs to templates.
-  enableCRDTemplates: false
\ No newline at end of file
+  enableCRDTemplates: false
+
+  # Validation webhook configuration url
+  # For example: https://$remotePilotAddress:15053/validate
```

Comment:
Wait, this port is in use by DNS-over-TLS ( 53 is the normal DNS port ). 

Can you use another one ? Confusion. 

I think 15017 is the container port in Istiod - let's use that.

Reply:
good point, I forgot webhook is running on container port 15017.

Reply:
updated